docs/containers/overview.md

```diff
@@ -59,7 +59,7 @@ You can run Docker commands to manage [images](https://docs.docker.com/engine/re
 
 ## Using image registries
 
-You can display the content and push/pull/delete images from [Docker Hub](https://hub.docker.com/) and [Azure Container Registry](https://docs.microsoft.com/azure/container-registry/):
+You can display the content and push/pull/delete images from [Azure Container Registry](https://docs.microsoft.com/azure/container-registry/), [Docker Hub](https://hub.docker.com/), [GitLab](https://gitlab.com/), and more:
```

Comment:
```suggestion
You can display the content and push/pull/delete images from [Azure Container Registry](https://docs.microsoft.com/azure/container-registry/), [Docker Hub](https://hub.docker.com/), [GitLab](https://gitlab.com/), and any other container registry:
```

Reply:
I hesitate to say "any" because it's far from any. The only other option is Registry V2 compliant registries that accept either password auth, or OAuth that starts with password auth.